modules/acquia_cms_site_studio/acquia_cms_site_studio.module

```diff
@@ -206,6 +207,67 @@ function acquia_cms_site_studio_modules_installed($modules, $is_syncing) {
         'administer style_guide',
       ]);
     }
+    if (in_array('node_revision_delete', $modules)) {
+      $config_handler = \Drupal::classResolver(ConfigHandlerFacade::class);
+      $config_handler->setModuleName('node_revision_delete');
+      // Node revision default config settings.
+      $default_settings = [
+        'defaults' => [
+          'amount' => [
+            'status' => TRUE,
+            'settings' => [
+              'amount' => 50,
+            ],
+          ],
+          'created' => [
+            'status' => TRUE,
+            'settings' => [
+              'age' => 12,
+            ],
+          ],
+          'drafts' => [
+            'status' => TRUE,
+            'settings' => [
+              'age' => 12,
+            ],
+          ],
+        ],
+      ];
```

Comment:
```suggestion
      ];
```
@apathak18 can this code be used in a method rather than calling this in multiple places.

Reply:
No as these are the settings related to only node revision delete and facade we created for processing the config settings and third party settings in a generic way.
For basically using the facade to pass the settings as a parameter so make the structure more robust.